src/Geta.404Handler/Controllers/NotFoundRedirectController.cs

```diff
@@ -119,22 +120,22 @@ public ActionResult Administer()
         }
 
         [ValidateInput(false)]
-        public ActionResult Save(string oldUrl, string newUrl, string skipWildCardAppend, int? pageNumber, int? pageSize)
+        public ActionResult Save(string oldUrl, string newUrl, string skipWildCardAppend, int redirectType, int? pageNumber, int? pageSize)
```

Comment:
It should be possible to have `redirectType` parameter to be an enum `RedirectType`.

Reply:
Done